src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -929,80 +496,14 @@ def main_tests(geo_redundant_backup):
                      F"--key {key['key']['kid']} --identity {identity['id']}" if database_engine == 'postgres' else ''
             ), checks=main_checks).get_output_in_json()
 
-            if geo_redundant_backup:
```

Comment:
there is no condition for postgre scenario here.

Reply:
Keep the if geo_redundant_backup: as is. When we call main_tests(true) for PG this will work. CUrrently we are calling main_tests(false) only for PG and hence I do not want you to remove the code

Reply:
Make sense, revert it